<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sidePanel">
     <item>
      <widget class="QPushButton" name="addIncome">
       <property name="toolTip">
        <string>Enter a new source of income.</string>
       </property>
       <property name="text">
        <string>Add Income</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addExpense">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Add Expense</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewExpenses">
       <property name="text">
        <string>View Expenses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewIncome">
       <property name="text">
        <string>View Income</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewStats">
       <property name="text">
        <string>View Stats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewCharts">
       <property name="text">
        <string>View Charts</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="panelSeparator">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>10</y>
      <width>16</width>
      <height>749</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="mainPanelWidget">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>10</y>
      <width>1036</width>
      <height>749</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="budgetViewWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QTableView" name="budgetView">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>1</y>
        <width>1021</width>
        <height>731</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QWidget" name="widget2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
